samples/sleep-then-completes-successfully.json

```diff
@@ -0,0 +1,344 @@
+{
```

Comment:
There's also an activity here, please give a name that reflects what this sample is for or create another sample that only sleeps.
Alternatively, just delete this sample if it doesn't provide much value.